BooruSharp/Booru/Yandere.cs

```diff
@@ -1,10 +1,18 @@
 ﻿namespace BooruSharp.Booru
 {
+    /// <summary>
+    /// Yande.re.
+    /// <para>https://yande.re/post</para>
```

Comment:
Why the "/post" part?
You don't write it for the others booru

Reply:
You're right, I forgot to remove the `/post` part after I copied the URL from the address bar in my browser.